documentation-processor/src/main/java/com/koval/resolver/processor/documentation/core/DocFileRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.koval.resolver.processor.documentation.core;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+
+public class DocFileRepository {
+
```

Comment:
What the reason for creating this class? Why not just use new FileInputStream(fileName) instead of DocFileRepository.getFile(String fileName)?

Reply:
The naming is kinda muddled because DocFileRepository was created before FileParser, I need to work on that.
The idea was [that parser classes do not know what is the InputStream that they are working](https://stackoverflow.com/a/17531932) with, because they don't really care. That allows are to unit test them without using any files.

Reply:
Okay, it's acceptable in this case